libjulia/backends/evm/EVMCodeTransform.cpp

```diff
@@ -34,6 +35,25 @@ using namespace dev::solidity;
 
 using Scope = dev::solidity::assembly::Scope;
 
+void CodeTransform::subtractReferences(map<string, size_t> const& _subtrahend)
+{
+	for (auto const& ref: _subtrahend)
+	{
+		assertThrow(m_references.count(ref.first), OptimizerException, "");
+		assertThrow(m_references.at(ref.first) >= ref.second, OptimizerException, "");
```

Comment:
~~Don't we have specific exceptions here? Shouldn't reuse the `OptimizerException`.~~

It seems we are just using `solAssert` in this file.

Reply:
We could also introduce a new exception